vue-components/.eslintrc.js

```diff
@@ -45,6 +45,9 @@ module.exports = {
 		'@typescript-eslint/no-require-imports': 'off',
 		'@typescript-eslint/no-var-requires': 'off',
 
+		// duplicate with eslint's indent
+		'@typescript-eslint/indent': 'off',
```

Comment:
Wait, but shouldn't we turn off `indent` instead? At least that is how I understand [typescript-eslint/indent.md](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/indent.md). Or are we affected by the issue mentioned there?

Reply:
I overlooked this, you're right.